app/views/podcast_planner/_form_options.html.erb

```diff
@@ -0,0 +1,33 @@
+<section>
+  <div class="mb-4">
+    <h2 class="fw-bold">What time of day do you plan to publish your episodes?</h2>
+    <p>
+      Select the time of day in which episodes would automatically be set to
+      publish once the state is changed to scheduled. This will prevent you from
+      resetting the value on each individual episode.
+    </p>
+
+    <div class="col-lg-4">
+      <div class="form-floating">
+        <%= form.select :publish_time, time_options, {}, data: {action: "podcast-planner#preview"} %>
+        <%= form.label :publish_time %>
+      </div>
+    </div>
+  </div>
+
+  <div class="mb-4">
+    <h2 class="fw-bold">Audio Segments</h2>
+    <p>
+      Make sure you choose a template associated with your series
+      Podcast Info.
+      This will help us forecast ad inventory available in each episode.
+    </p>
+
+    <div class="col-lg-4">
+      <div class="form-floating">
+        <%= form.number_field :segment_count, {min: 1, max: 10, step: 1, value: 2, data: {action: "podcast-planner#preview"}} %>
```

Comment:
do we have a min/max audio segment count? i'm using 1 and 10 for now, but i feel like most don't go above 4? and  the highest "outlier" i've seen was maybe 6 or 7?

Reply:
This feels sane.  I can't imagine having more than 5 segments... but who knows.  Maybe someone wants just a ton of midroll ads.